tests/update_workflow_test.go

```diff
@@ -3082,6 +3082,7 @@ func (s *integrationSuite) TestUpdateWorkflow_NewScheduledSpeculativeWorkflowTas
 	// Close shard, Speculative WT with update will be lost.
 	s.closeShard(tv.WorkflowID())
 
+	// Ensure, there is no new WT.
```

Comment:
FWIW, I've more or less stopped commenting in tests and instead putting the comment as t.Log. Serves effectively the same purpose but might possibly also help as debugging output if the test fails. Not at all important for this PR, or ever but I kind of like it.